css/partner/partner.css

```diff
@@ -1 +1,3 @@
-@import url('shop-profile/shop-profile.css');
\ No newline at end of file
+@import url('shop-profile/shop-profile.css');
```

Comment:
LOOK. AT. THIS.

Did you not get the hint from this import statement?
See how it its clearly importing shop-profile.css, which is located inside shop-profile folder? You should've done the same to your page.